pages/task/list/views/tasklist.jsx

```diff
@@ -42,7 +42,7 @@ function TaskFilters({ hasTasks, progressOptions }) {
             label={<Snippet>filters.pplType.label</Snippet>}
             prop="pplType"
             formatter={filter => <Snippet>{`filters.pplType.options.${filter}`}</Snippet>}
-            append={['applications', 'amendments', 'revocations', 'transfers', 'continuations', 'hasDeadline', 'ra']}
+            append={['applications', 'amendments', 'revocations', 'transfers', 'continuations', 'hasDeadline', 'ra', 'changeLicenceHolder']}
```

Comment:
Can we move the new one left to between `transfers` and `continuations`?
```suggestion
            append={['applications', 'amendments', 'revocations', 'transfers', 'changeLicenceHolder', 'continuations', 'hasDeadline', 'ra']}
```

Reply:
That way the filters are grouped roughly as "task type filters" and "random other stuff"